### ISSUE TITLE
Mind

### ISSUE THREAD

**Issue by DongfeiJi** [tianweiy/CenterPoint#171]
Hi~I'd like to ask a question.
Below is get target func.
Why rotate the center point of gt_of_rois?
-----------------------------------------------------------------------
rois = targets_dict['rois']  # (B, N, 7 + C) torch.Size([4, 128, 7])
gt_of_rois = targets_dict['gt_of_rois']  # (B, N, 7 + C + 1)torch.Size([4, 128, 8])
targets_dict['gt_of_rois_src'] = gt_of_rois.clone().detach()

roi_ry = limit_period(rois[:, :, 6], offset=0.5, period=np.pi*2)#torch.Size([4, 128])

gt_of_rois[:, :, :6] = gt_of_rois[:, :, :6] - rois[:, :, :6]
gt_of_rois[:, :, 6] = gt_of_rois[:, :, 6] - roi_ry

gt_of_rois = box_torch_ops.rotate_points_along_z(
    points=gt_of_rois.view(-1, 1, gt_of_rois.shape[-1]), angle=-roi_ry.view(-1)
).view(batch_size, -1, gt_of_rois.shape[-1])#torch.Size([4, 128, 8])

**Comment by tianweiy** [comment]
Hi, which file does this code come from ?

**Comment by DongfeiJi** [comment]
> Hi, which file does this code come from ?

CenterPoint\det3d\models\roi_heads\roi_head_template.py
line 43

**Comment by DongfeiJi** [comment]
> see this #121

Thanks very much!
